nvflare/app_opt/xgboost/histogram_based_v2/controller.py

```diff
@@ -72,6 +75,9 @@ def __init__(
         Args:
             adaptor_component_id - the component ID of server target adaptor
             num_rounds - number of rounds
+            training_mode - Split mode (horizontal, vertical, horizontal_secure, vertical_secure)
+            xgb_params -The parameter passed to XGBoost
+            xgb_options - Other options for XGBoost
```

Comment:
what things belong to "xgb_params" vs xgb_options? what are their type?

Reply:
Updated the document.